etc/cyclonedds.xsd

```diff
@@ -44,17 +44,16 @@ CycloneDDS configuration</xs:documentation>
       <xs:attribute name="Id">
         <xs:annotation>
           <xs:documentation>
-&lt;p&gt;Domain id this configuration applies to, or "any" if it applies to all
-domain ids.&lt;/p&gt;&lt;p&gt;The default value is: &amp;quot;any&amp;quot;.&lt;/p&gt;</xs:documentation>
+&lt;p&gt;Domain id this configuration applies to, or "any" if it applies to all domain ids.&lt;/p&gt;
```

Comment:
Shouldn't the " characters be turned into &quot;? (Maybe no-one cares, and certainly the original &amp;quot; was broken.)

Reply:
I translated all xml entities at first, but `trang` didn't replace the ones in the comment, so I figured it'd be more readable like this. So it isn't required, but I can reinstate if desired.

Reply:
If a plain double-quote character works, it is good enough for me :)